lib/jbuild

```diff
@@ -0,0 +1,11 @@
+(library
+ ((name ezxenstore)
+  (public_name ezxenstore)
+  (modules (xenstore watch))
+  (wrapped false)
```

Comment:
Ah I see, so this is for compatibility, to not break code that uses this library by putting everything under one module that previously was top-level.